pages/config.py

```diff
@@ -0,0 +1,54 @@
+import PySimpleGUI as sg
+import datetime
+import configparser
+
+current_year = datetime.datetime.now().year
+previous_year = current_year - 1
+
+layout = [
+    [sg.Text("Dados Pessoais")],
+    [sg.Text("CNPJ:"), sg.Input(size=(14, 1), key="-CNPJ-", enable_events=True)],
+    [sg.Text("Gerar DAS")],
+    [sg.Text("Mês:"), sg.Combo(['Janeiro', 'Fevereiro', 'Março', 'Abril', 'Maio', 'Junho', 'Julho', 'Agosto', 'Setembro', 'Outubro', 'Novembro', 'Dezembro'], key="-MONTH-", enable_events=True)],
```

Comment:
Ideia de função:

def returnMonth(month=None):
    months = ['Janeiro', 'Fevereiro', 'Março', 'Abril', 'Maio', 'Junho', 'Julho', 'Agosto', 'Setembro', 'Outubro', 'Novembro', 'Dezembro']
    if month is None:
        return meses
    else:
        return months.index(month) + 1